support/installing-protoc.mdx

```diff
@@ -2,28 +2,57 @@
 title: "Installing Protoc"
 ---
 
-Since `v0.12.0` `protoc` is a requirement to use and install `cargo-shuttle`. To install `protoc`, here
-are some instructions for Windows, Linux and Mac:
+Since shuttle v0.12.0, `protoc` is a requirement to install `cargo-shuttle` and to use `cargo-runtime`.
+Here are some instructions for installing `protoc` on Windows, Linux and Mac:
 
 ### Windows
 
 On Windows the simplest way to install `protoc` is to first install [chocolatey](https://chocolatey.org/install), 
 and then simply run: `choco install protoc`.
 
+Then, verify that it's working:
+
+```bash
+# should show a version number
+protoc --version
+```
+
+**Note:** *You might need to restart your shell.
```

Comment:
Perhaps we could link to this: https://docs.chocolatey.org/en-us/troubleshooting#my-path-is-not-getting-updated as well?